packages/Button/src/Button.js

```diff
@@ -43,7 +43,7 @@ const propTypes = {
   isSubmit: PropTypes.bool,
 
   /** The visual style of the button. */
-  kind: PropTypes.oneOf(Kinds),
+  kind: PropTypes.oneOf(Kinds.ALL),
```

Comment:
I think we are going to change this approach later right @mikrotron ?

Reply:
I like the idea of having a `types` package that can be consumed by our components, but also by consuming apps to make it easier to use typed props.  

We should probably open a "proposal" issue and get a little feedback before we assume we _will_ change this, but it's probably a safe bet.